recipes/moz-sql-parser/meta.yaml

```
@@ -0,0 +1,39 @@
+{% set name = "moz-sql-parser" %}
+{% set version = "2.49.19205" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: dd8fdd447d7e31c51e1a65d997929b6ebe1884e9eef25ea190ed5784a5124eb7
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - mo-future >=2.48.19205
+    - pyparsing ==2.3.1
+    - python
+
+test:
+  imports:
+    - moz_sql_parser
+
+about:
+  home: "https://github.com/mozilla/moz-sql-parser"
+  license: "Mozilla Public 2.0 (MPL 2.0)"
```

It's nice to use the SPDX identifiers to try and make the metadata consistent across conda-forge:
```suggestion
  license: MPL-2.0
```